models/src_dynamics_365_crm.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+
+sources:
+  - name: dynamics_365_crm 
+    database: "{% if target.type not in ('spark') %}{{ var('dynamics_365_crm_database', target.database) }}{% endif %}"
+    schema: "{{ var('dynamics_365_crm_schema', 'dynamics_365')}}"
+
+    loader: fivetran
+    loaded_at_field: _fivetran_synced
+      
+    freshness:
+      warn_after: {count: 72, period: hour}
+      error_after: {count: 96, period: hour}
+
+    tables:
+      - name: account
+        identifier: "{{ var('dynamics_365_crm_account_identifier', 'account')}}"
```

Comment:
Because all of these sources can be disabled, we should add the config to turn off the sources if the corresponding variable is disabled.

Reply:
Thank you. Added.